### ISSUE TITLE
Utilities.CreateEntityByName causes exception after updating to v76

### ISSUE THREAD

**Issue by Dliix66** [roflmuffin/CounterStrikeSharp#141]
```
CBeam beam = Utilities.CreateEntityByName<CBeam>("beam");
```

This was working fine before I updated to v76, now causing this issue:

```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> CounterStrikeSharp.API.Core.NativeException: Invalid function pointer
   at CounterStrikeSharp.API.Core.ScriptContext.CheckErrors() in /home/runner/work/CounterStrikeSharp/CounterStrikeSharp/managed/CounterStrikeSharp.API/Core/ScriptContext.cs:line 176
   at CounterStrikeSharp.API.Core.NativeAPI.ExecuteVirtualFunction[T](IntPtr function, Object[] arguments) in /home/runner/work/CounterStrikeSharp/CounterStrikeSharp/managed/CounterStrikeSharp.API/Core/API.cs:line 893
   at CounterStrikeSharp.API.Utilities.CreateEntityByName[T](String name) in /home/runner/work/CounterStrikeSharp/CounterStrikeSharp/managed/CounterStrikeSharp.API/Utilities.cs:line 48
```

If I recall, I was on v63 before updating and it was working properly.

**Comment by roflmuffin** [comment]
I think the sig is just bad? Seems unlikely the change from 63 to 76 would have broken this as the core sig scanning library was untouched and I tried replicating it locally using the old managed code and I get the error on latest cs version

**Comment by Dliix66** [comment]
@roflmuffin I checked, i was on v64 (https://github.com/roflmuffin/CounterStrikeSharp/actions/runs/6940234182) and I confirm it is working on this release

**Comment by Dliix66** [comment]
Update: Working good on v72

**Comment by mificot** [comment]
@Dliix66 i think it will be resolved here https://github.com/roflmuffin/CounterStrikeSharp/issues/123

**Comment by Dliix66** [comment]
Nice thanks! Sorry i didnt see this earlier!

**Comment by KillStr3aK** [comment]
this can be closed as #123 is now resolved.
